web/geosearch/datapunt_geosearch/registry.py

```diff
@@ -117,7 +119,7 @@ def init_dataset(self, row, class_name, dsn_name,
         """
         from datapunt_geosearch.datasource import DataSourceBase
         if field_name_transformation is None:
-            field_name_transformation = lambda x: x
+            field_name_transformation = lambda x: to_snake_case(x)
```

Comment:
Dit is niet nodig, omdat het kan probleemen met VSD brengen.

Reply:
Is goed, ik haal em weg